assets/javascripts/templates/connectors/user-custom-preferences/region.hbs

```diff
@@ -3,6 +3,7 @@
   <div class="controls">
     {{region-input
       value=model.custom_fields.holidays-region
+      allowNoneRegion=true
```

Comment:
This is entirely optional, but you could also initialize `allowNoneRegion: true` (or false) in the JS file of the component, and that would mean that you wouldn't need to add it as a parameter to each `region-input` instance in the template.

Reply:
Good suggestion! I made `false` the default, so it will only display holiday regions that the Holiday gem supports by default. :smiley: